### PR TITLE
:construction: Update: Add 98th CatchUp Summary

### PR DESCRIPTION
**Pull request by dheerajdlalwani**

### CODE REVIEW

File: summary/sessions/098/content.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Date: 24-09-2022
+
+Duration: 4 hrs 2 mins
+
+==== Topics Discussed
+
+* link:https://twitter.com/harshgkapadia[Harsh Kapadia^] talked about new challenges that he is currently facing in US. 
+    ** He talked about SQL stuff, Tic tac toe game in java, extensible, order and chaos. He said assignments were challenging but fun and good. He also discussed how he used sockets to test throughput using java.
```
Comment (tusharnankani) [inline summary/sessions/098/content.adoc:8]: - SQL stuff?
- Tic-tac-toe
- extensible, order, chaos? Context?
- He said that the*

File: summary/sessions/098/content.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Date: 24-09-2022
+
+Duration: 4 hrs 2 mins
+
+==== Topics Discussed
+
+* link:https://twitter.com/harshgkapadia[Harsh Kapadia^] talked about new challenges that he is currently facing in US.
```
Comment (tusharnankani) [inline summary/sessions/098/content.adoc:7]: in the* US

File: summary/sessions/098/content.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Duration: 4 hrs 2 mins
+
+==== Topics Discussed
+
+* link:https://twitter.com/harshgkapadia[Harsh Kapadia^] talked about new challenges that he is currently facing in US. 
+    ** He talked about SQL stuff, Tic tac toe game in java, extensible, order and chaos. He said assignments were challenging but fun and good. He also discussed how he used sockets to test throughput using java.
+    ** We talked about digital communication, how it is made secure using the link:https://en.wikipedia.org/wiki/Public-key_cryptography[public key cryptography^] and how link:https://en.wikipedia.org/wiki/Public_key_certificate[digital certificates^] help with that.
+    ** We then discussed how link:https://en.wikipedia.org/wiki/Transport_Layer_Security[TLS^] and link:https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol[OCSP^] hits the certificate authority to verify the certificate.
+        *** link:https://en.wikipedia.org/wiki/Certificate_revocation_list[CRL - Certificate revocation list^] checks for the certificate locally and if it exists  then it does not make a request.
```
Comment (tusharnankani) [inline summary/sessions/098/content.adoc:11]: exists < space > then

File: summary/sessions/098/content.adoc [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        *** link:https://networking.harshkapadia.me/cryptography[More information^]
+* link:https://www.google.com/intl/en_in/chrome[Google Chrome^] will stop validating link:https://en.wikipedia.org/wiki/Public_key_certificate[digital certificates^].
+* link:https://twitter.com/clumsy_coder[Kaushal Joshi^] talked about blogging, freelance, internship, active search.
+* link:https://www.linkedin.com/in/anilharwani[Anil Harwani^] started a discussion on the ability to focus while learning new things.
+    ** We are supposed to be studying right now so that we don't regret. 
+    ** link:https://www.linkedin.com/in/anilharwani[Anil Harwani^] also pointed out how everything is a trade off and nothing is free even if it says free.
+    ** Time is extremely precious, it won't come back.
+    ** Every decision is a trade-off.
+    ** When we have stuff to do then why should we waste time? We should keep an agenda.
```
Comment (tusharnankani) [inline summary/sessions/098/content.adoc:20]: we have things* to do
Note: Avoid using stuff

File: summary/sessions/098/content.adoc [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+* link:https://www.linkedin.com/in/anilharwani[Anil Harwani^] shared how he used to optimise his time while he was studying in college. He used to challenge himself or read a book while he used to travel.
+    ** You need to hold yourself accountable since no one else will.
+* link:https://twitter.com/harshgkapadia[Harsh Kapadia^] talked about how he feels about passion being missing when students complete assignments. He feels most of them are more driven towards having an internship or a job.
+    ** Most Indian students do jobs or study engineering since it is a means to an end.
+    ** Nine times out of ten, a job is a means to an end and that's why it's called a job.
+* We discussed ways in which a processor can go idle to save power.
+    ** link:https://en.wikipedia.org/wiki/Clock_gating[Clocked Gating^]. Clock gating is a power-saving feature in semiconductor microelectronics that enables switching off circuits.
+    ** link:https://en.wikipedia.org/wiki/Power_gating[Power Gating^]. Power gating is a technique used in integrated circuit design to reduce power consumption, by shutting off the current to blocks of the circuit that are not in use.
```
Comment (tusharnankani) [inline summary/sessions/098/content.adoc:30]: to block*

File: summary/sessions/098/content.adoc [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* We talked about how foundational courses that colleges include in syllabus are very important to become a good software engineer.
+    ** Many people have lost the ability to dive deep into a topic and understand the crux of it.
+    ** Documenting ways to implement a project is not helpful for others since many people would then start building the same project even though they aren't qualified to build it. This is a problem since people who do not understand whats happening will then blame the creator.
```
Comment (tusharnankani) [inline summary/sessions/098/content.adoc:33]: what's* happening

**Review by tusharnankani — COMMENTED** [review]
Rest LGTM

File: summary/sessions/098/content.adoc (hunk 2 above)
Comment (PranavDani) [reply summary/sessions/098/content.adoc:7]: 👍

File: summary/sessions/098/content.adoc (hunk 1 above)
Comment (PranavDani) [reply summary/sessions/098/content.adoc:8]: Thank you for pointing it out, I did not read this one.

File: summary/sessions/098/content.adoc (hunk 4 above)
Comment (PranavDani) [reply summary/sessions/098/content.adoc:20]: I never use ```stuff```, @dheerajdlalwani?

File: summary/sessions/098/content.adoc (hunk 5 above)
Comment (PranavDani) [reply summary/sessions/098/content.adoc:30]: ?

File: summary/sessions/098/content.adoc (hunk 6 above)
Comment (PranavDani) [reply summary/sessions/098/content.adoc:33]: 👍

File: summary/sessions/098/content.adoc (hunk 5 above)
Comment (PranavDani) [reply summary/sessions/098/content.adoc:30]: blocks is correct in this context (Imagine building blocks ~ (Circuit blocks))

File: summary/sessions/098/content.adoc (hunk 3 above)
Comment (PranavDani) [reply summary/sessions/098/content.adoc:11]: 👍

**Pull request merged**